src/Core/src/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -1975,7 +1975,7 @@ override Microsoft.Maui.Font.ToString() -> string!
 override Microsoft.Maui.FontImageSourceService.GetDrawableAsync(Microsoft.Maui.IImageSource! imageSource, Android.Content.Context! context, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Microsoft.Maui.IImageSourceServiceResult<Android.Graphics.Drawables.Drawable!>?>!
 override Microsoft.Maui.FontImageSourceService.LoadDrawableAsync(Microsoft.Maui.IImageSource! imageSource, Android.Widget.ImageView! imageView, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Microsoft.Maui.IImageSourceServiceResult?>!
 override Microsoft.Maui.Graphics.MauiDrawable.Dispose(bool disposing) -> void
-override Microsoft.Maui.Graphics.MauiDrawable.OnBoundsChange(Android.Graphics.Rect? bounds) -> void
+override Microsoft.Maui.Graphics.MauiDrawable.OnBoundsChange(Android.Graphics.Rect! bounds) -> void
```

Comment:
This is fine for nullability from base SDKS as they are not actual changes but just warnings at compile time - and we cannot control it.